runners/flink/src/main/java/org/apache/beam/runners/flink/translation/wrappers/streaming/io/source/FlinkSource.java

```diff
@@ -73,18 +71,6 @@ public static <T> FlinkUnboundedSource<T> unbounded(
     return new FlinkUnboundedSource<>(stepName, source, serializablePipelineOptions, numSplits);
   }
 
-  public static FlinkUnboundedSource<byte[]> unboundedImpulse(long shutdownSourceAfterIdleMs) {
```

Comment:
this change seems unrelated to the PR, please revert or move it to separate commit

Reply:
we squash commits on merge requests, so moving to different commit will be a nop. The method is unused, the removal is pure cleanup.

Reply:
🤦